app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 comfort_class: int,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one